### PR TITLE
[ZPI-151] Add map integration

### PR DESCRIPTION
**Pull request by bartoszgableta**

### CODE REVIEW

File: app/components/TransferPointNode.tsx [hunk 1]
```diff
@@ -88,4 +150,16 @@ const createStyles = (theme: MD3ThemeExtended) =>
       left: "100%",
       height: 20,
     },
+    leftComponent: {
+      position: "absolute",
+      top: "50%",
+      transform: [{ translateY: -22 }],
+      right: "115%",
```
Comment (AdrianBroniecki) [inline app/components/TransferPointNode.tsx:157]: Why so unusual values?

File: app/components/TransferPointNode.tsx [hunk 2]
```diff
@@ -88,4 +150,16 @@ const createStyles = (theme: MD3ThemeExtended) =>
...
+      display: "flex",
+      alignItems: "center",
+    },
+    leftComponentButton: {
+      margin: 0,
```
Comment (AdrianBroniecki) [inline app/components/TransferPointNode.tsx:162]: Isn't margin 0 by default?

File: app/components/TransferPointNode.tsx (hunk 1 above)
Comment (bartoszgableta) [reply app/components/TransferPointNode.tsx]: Fixed

File: app/components/TransferPointNode.tsx (hunk 2 above)
Comment (bartoszgableta) [reply app/components/TransferPointNode.tsx]: It isn't :P but i refactored it a little bit now it should work better

File: app/components/TransferPointNode.tsx [hunk 3]
```diff
@@ -26,20 +34,34 @@ const TRANSFER_TYPE_MAP = {
...
 
+// For Google Maps travel mode
+const TRANSFER_TYPE_MAP_GOOGLE = {
+  [TransferType.Car]: "driving",
+  [TransferType.Motorbike]: "driving",
+  [TransferType.Bicycle]: "bicycling",
+  [TransferType.Walk]: "walking",
+  null: "driving",
+};
+
```
Comment (AdrianBroniecki) [inline app/components/TransferPointNode.tsx:45]: Maybe move to some consts file?

**Review by AdrianBroniecki — APPROVED** [review]

**Pull request merged**